### ISSUE TITLE
Muted for reporting DamageMeters in /raid

### ISSUE THREAD

**Issue by ifyousayso** [elysium-project/server#1753]
Really? One minute timeout? Not blizzlike at all.

**Comment by Haiwuu** [comment]
You got caught by the spam filter, which is unfortunately needed in 2017. Please close.

**Comment by lovemonkeyz** [comment]
The spam filter really should be tuned better then.

**Comment by ifyousayso** [comment]
I still think this is an issue. It's understandable if one gets muted for spamming in a global channel (and perhaps even /yell or /say), but private ones should be ignored. This is ridiculous..

**Comment by Phoosy** [comment]
Private messages are the prime candidate for spammers, your suggestion is that they be _less_ moderated by the spam filter?

**Comment by Phoosy** [comment]
I would agree that /guild, /raid, etc. should probably be ignored, but that would be pending a major overhaul of the AS to deal with the ridiculous gold spam we've been getting hit with anyway.

**Comment by ifyousayso** [comment]
Oh, sorry, my bad. I wrote a tiny addon that has filtered out all seller messages for the past 2 months or so.. forgot about the whispers.

Unfortunately, I don't want to make it public since that would give them the opportunity to adapt.

1. I have never received more than one seller message from a specific character at a time.
2. Some sites whisper in waves, but from different characters.
3. While there are (raid) addons that whisper the same target simultaneously, those only do so to other raid members.
4. Instead of muting, a filter can be set up to detect a specific pattern and as soon as a character is caught, the entire account should be suspended (along with all other accounts used from the same IP address) until someone complains.. and if/when they do, point to the fact that they behave exactly like bots.
5. I (personally) would even apply the filter to global channels since the sender is either a seller or a troll. Breaking this rule can result in a 24h (channel) mute and breaking it a second time = ban.

With this, one whisper and a bunch of accounts are disabled.

.. but then I have also ignored more than 800 characters because of trolling in World, so this solution might not be for everyone. :)

**Comment by ssishtar** [comment]
The spam filter is way too powerful, since it affects all channels and it shouldnt. For example, my guild is using Raid Calendar, which as you know, uses a separate invisible channel to communicate. The Elysium server is muting as randomly tens of times a day because of the addon, which is idiotic. 

TLDR: The spam filter SHOULD act ONLY on public channels like General, LFG, Trade etc. It shouldn't act on whispers, on raids channels, party channels, guild channels and so on. These channels can be moderated by players IN GAME, who don't need any help. A Guild master can mute in seconds or even gkick, a party/raid leader can dispose of an eventual moron also. 

Please Elysium, keep the spam filter on general channels, but REMOVE it asap on raid/party/player made channels.
